sst/config/a64fx-config.py

```diff
@@ -0,0 +1,187 @@
+import sst
+
+DEBUG_L1 = 0
+DEBUG_L2 = 0
+DEBUG_MEM = 0
+DEBUG_LEVEL = 0
+
+
+# ------------------------------------------------ Utility -------------------------------------------
+
+def getMemoryProps(memory_size: int, si: str):
+      props = {
+            "start_addr": 0,
+            "end_addr": 0,
+            "size": ""
+      }
+      props["size"] = "%s%s" % (memory_size , si)
+      if si == "GiB":
+            props["end_addr"] = memory_size * 1024 * 1024 * 1024 - 1
+      elif si == "MiB":
+            props["end_addr"] = memory_size * 1024 * 1024 - 1
+      elif si == "KiB":
+            props["end_addr"] = memory_size * 1024 - 1
+      elif si == "B":
+            props["end_addr"] = memory_size - 1
+      else:
+            raise Exception("Unknown SI units provided to getMemoryProps")
+      return props
+
+# ------------------------------------------------ Utility -------------------------------------------
+
+
+
+# ------------------------------------------- A64FX Properties ---------------------------------------
+
+# This SST configuration file represents the memory model for the Fujitsu A64fx processor.
+# Reference: https://github.com/fujitsu/A64FX/blob/master/doc/A64FX_Microarchitecture_Manual_en_1.8.pdf
+
+# Cache line size of L1 & L2 in A64FX in bytes.
+A64FX_CLW = 256
+# Clock Frequency of A64FX.
+A64FX_CLOCK = "2GHz"
```

Comment:
Given that we compare to the Isambard A64FX which runs at 1.8GHz, I think this value should reflect this implementation

Reply:
Done.